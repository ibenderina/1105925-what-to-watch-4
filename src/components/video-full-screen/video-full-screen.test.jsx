import VideoFullScreen from "@components/video-full-screen/video-full-screen";

it(`Should VideoFullScreen render correctly`, () => {
  const commonNumber = 1;

  let tree;
  window.act(() => {
    tree = window.create(<VideoFullScreen
      history={{goBack: () => {}}}
      match={{params: {filmId: commonNumber}}}
    />);
  });
  expect(tree.toJSON()).toMatchSnapshot();
});

