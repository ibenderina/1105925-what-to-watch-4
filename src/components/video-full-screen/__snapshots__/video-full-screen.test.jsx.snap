// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoFullScreen render correctly 1`] = `
<h2>
  Loading...
</h2>
`;
